priv/repo/migrations/20210608124110_message_media_trigger.exs

```diff
@@ -0,0 +1,41 @@
+defmodule Glific.Repo.Migrations.MessageMediaTrigger do
+  use Ecto.Migration
+
+  def up do
+    message_media_trigger()
+  end
+
+  defp message_media_trigger do
+    execute """
+    CREATE OR REPLACE FUNCTION update_message_updated_at()
+      RETURNS trigger AS $$
+
+      BEGIN
+      CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+        IF (TG_OP = 'UPDATE') THEN
+
+          UPDATE messages set updated_at = (CURRENT_TIMESTAMP at time zone 'utc') where media_id = NEW.id;
+        ELSE
+          IF (TG_OP = 'DELETE') THEN
+            UPDATE messages set updated_at = (CURRENT_TIMESTAMP at time zone 'utc') where media_id = old.id;
```

Comment:
Are we allowing to delete message media? 
also in case, it's deleted it will remove the URL from the bigquery also.

Let's remove this.

Reply:
Also since you are triggeing this on update only this function will never be called.